model/geographyHomepage/model.go

```diff
@@ -0,0 +1,20 @@
+package geographyHomepage
+
+import "github.com/ONSdigital/dp-frontend-models/model"
+
+// Page contains the data re-used on each page as well as the data for the current page
+type Page struct {
+	model.Page
+	Data GeographyList `json:"data"`
+}
+
+// GeographyList represents the data on the Geography list page
+type GeographyList struct {
+	Items []GeographyDataType `json:"items"`
```

Comment:
Could we name this something more specific, like `AreaTypes`? Items is quite generic, is we start adding more to the model is isn't necessarily clear what it contains?